client/src/components/LandingPage/LandingPage.tsx

```diff
@@ -61,6 +61,7 @@ const LandingPage: FC<LandingPageProps> = ({ testSuites }) => {
                     <ListItem
                       key={testSuite.id}
                       button
+                      data-testid="testing-suite-option"
```

Comment:
Does this get automatically stripped from real builds?

Reply:
No, but is there a downside to leaving it in the build? It is in some other components already

Reply:
It's not a big deal, I just thought it might be and wanted to check.